CHANGELOG.md

```diff
@@ -21,21 +23,21 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Fix monitored file system mount points.
 
-## [v0.3.0]
+## [0.3.0] 2019-05-24
```

Comment:
I fixed the links and added the dates according to the spec.
You can e.g. see the changes for this release here https://github.com/giantswarm/kubernetes-node-exporter/compare/v0.2.0...v0.3.0.